### PR TITLE
[skip ci] - tdz documentation for block-scoping

### PR DESCRIPTION
**Pull request by rajzshkr**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | https://github.com/babel/babel/issues/527#issuecomment-256357224
| Patch: Bug Fix?          |
| Major: Breaking Change?  |
| Minor: New Feature?      |
| Tests Added + Pass?      | No
| Documentation PR         | 
| Any Dependency Changes?  | No
| License                  | MIT

Documentation for `tdz` in block-scoping

### CODE REVIEW

File: packages/babel-plugin-transform-block-scoping/README.md [hunk 1]
```diff
@@ -81,3 +83,13 @@ for (let i = 0; i < 5; i++) {
 ```
 
 In extremely performance-sensitive code, this can be undesirable. If `"throwIfClosureRequired": true` is set, Babel throws when transforming these patterns instead of automatically adding an additional function.
+
+`tdz`
```
Comment (existentialism) [inline packages/babel-plugin-transform-block-scoping/README.md]: Nit: can we set each option as a heading and add its type and default value ([example](https://github.com/babel/babel/tree/master/packages/babel-preset-react#development))

> ## Options
> 
> ### `throwIfClosureRequired `
> `boolean`, defaults to `false`.
> <...>
>
> ### `tdz`
> `boolean`, defaults to `false`.
> <...>

**Comment by existentialism** [comment]
@rajzshkr actually, it looks like https://github.com/babel/babel/pull/6782/files adds documentation for the same option, so let's go with that?

@maurobringolf mind including my tweaks to your PR?

**Comment by rajzshkr** [comment]
@existentialism yup.

**Comment by maurobringolf** [comment]
@existentialism Yes, done

**Pull request closed without merging**